tests/operators/test_bash_operator.py

```diff
@@ -33,6 +34,15 @@
 
 
 class BashOperatorTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        configuration.conf.set('core', 'default_task_retries', "3")
```

Comment:
I think you can use https://github.com/apache/airflow/blob/577f8936727454eb4ff54588cd525c5244ef6d52/tests/test_utils/decorators.py#L28 instead